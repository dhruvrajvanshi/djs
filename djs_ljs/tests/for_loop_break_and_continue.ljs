
const c = @builtin("c_str")
function main(): @builtin("c_int") {
    for (let i: @builtin("c_int") = 0; i < 10; i++) {
        putchar(i + 48)
        puts(c` LOOP`)
        if (i < 3) {
            puts(c` < 3`)
            continue
        }
        if (i > 5) {
            puts(c` BREAKING`)
            break
        }
    }
}
extern function puts(msg: *@builtin("c_char")): @builtin("c_int")
extern function putchar(c: @builtin("c_int")): @builtin("c_int")
