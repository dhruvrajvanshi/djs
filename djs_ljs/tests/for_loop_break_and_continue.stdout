0 LOOP
 < 3
1 LOOP
 < 3
2 LOOP
 < 3
3 LOOP
4 LOOP
5 LOOP
6 LOOP
 BREAKING
